rustyms/src/identification/instanovo.rs

```diff
@@ -52,7 +52,13 @@ impl From<InstaNovoData> for IdentifiedPeptide {
     fn from(value: InstaNovoData) -> Self {
         Self {
             score: Some(2.0 / (1.0 + 1.01_f64.powf(-value.score))),
-            local_confidence: Some(value.local_confidence.iter().map(|v| *v / 100.0).collect()),
+            local_confidence: Some(
+                value
+                    .local_confidence
+                    .iter()
+                    .map(|v| 2.0 / (1.0 + (-v).exp()))
```

Comment:
Is it intended for this calculation to differ from the one above (line 54 in my view) where the full score is normalised?

Reply:
Yes this is intentional, when I use the other calculation you get very misrepresented values. The current implementation represents the full range better.